cli/lib/cli.js

```diff
@@ -93,6 +93,7 @@ const parseVariableOpts = (fnArgs, args) => {
 }
 
 const descriptions = {
+  autoCancelAfterFailures: 'sets auto cancellation for the run. pass a number for the failed test threshold, or "false"',
```

Comment:
Should this be a configuration option as well so users don't have to pass this as a flag each time?

Reply:
We need to specify this only applies when recording to the dashboard. Also, is this number of specs that failed or total failed tests? If this run was split across multiple machines, is this total per machine or per run?

Reply:
This should be the number of tests that failed per run

cc. @ryanpei

Reply:
Def want to tie it back to record mode though, otherwise I'd expect this to work locally when I wasn't recording.